bundles/AdminBundle/src/Controller/Traits/UserNameTrait.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\AdminBundle\Controller\Traits;
+
+use Pimcore\Model\User;
+
+/**
+ * @internal
+ */
```

Comment:
Why `internal`? Seems like a useful Trait which could be used elsewhere.

Reply:
Agree, we should keep that one `@internal`.